test/.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+  "extends": "eslint:recommended",
```

Comment:
Is there a reason the tests directory needs its own eslint config file? And if so, shouldn't it extend the config file in the parent directory to avoid duplicating so much of that file? Or is that not possible?

Reply:
Been wondering about this myself.  

Looking into it.  Stand by…